controllers/drcluster_controller.go

```diff
@@ -141,6 +145,24 @@ func (r *DRClusterReconciler) drClusterConfigMapMapFunc(configMap client.Object)
 	return requests
 }
 
+func (r *DRClusterReconciler) drClusterSecretMapFunc(secret client.Object) []reconcile.Request {
+	if secret.GetNamespace() != NamespaceName() {
```

Comment:
Not clear what is this namespace. Not an issue in your change but maybe it is time to make this more clear.